Added note that min-block-occupancy-ratio is ignored for PoS (#1354)
hyperledger/besu-docs: versioned_docs/version-23.4.1/public-networks/reference/cli/options.md

Minimum occupancy ratio for a mined block if the transaction pool is not empty. When filling a block during mining, the occupancy ratio indicates the threshold at which the node stops waiting for smaller transactions to fill the remaining space. The default is 0.8.

:::note

> The `--min-block-occupancy-ratio` option is currently ignored for PoS (Proof of Stake) networks in Hyperledger Besu. This means that setting this option will not have any effect when running Besu with a PoS consensus mechanism.

Reviewer: ```suggestion
Besu ignores the `--min-block-occupancy-ratio` option for proof of stake networks (for example, Mainnet).
```
